src/renderers/dom/shared/__tests__/ReactServerRendering-test.js

```diff
@@ -549,4 +661,27 @@ describe('ReactDOMServer', () => {
       );
     }).toThrowError(/Cannot assign to read only property.*/);
   });
+
+  describe('basic rendering', function() {
+    itRenders('should render a blank div', render =>
```

Comment:
nit: can we rename this function or the tests so that "it renders should render a blank div" is an actual sentence/phrase?